packages/sdk/src/base/auth.ts

```diff
@@ -72,7 +74,10 @@ export class Auth extends IAuth {
 
 		if (this.timer) clearTimeout(this.timer);
 
-		const msWaitUntilRefresh = this._storage.auth_expires - this.msRefreshBeforeExpires;
+		const msWaitUntilRefresh = Math.max(
+			this._storage.auth_expires - this.msRefreshBeforeExpires,
+			this.msRefreshBeforeExpires
+		);
```

Comment:
@rijkvanzanten is this still prone to issues? Basically when we set `ACCESS_TOKEN_TTL` to `10s` for example, but `msRefreshBeforeExpires` is higher (defaults to `30000ms`/`30s`), doing simple subtraction ended up making it negative thus causes infinite loop of refreshes. This is why the use of Math.max here, but not sure if it's too rudimentary or not taking edge cases into account, if any. Or perhaps it should be "use storage auth expires ONLY without subtracting when it's already lower than msRefreshBeforeExpires".

Reply:
This is now obsolete after #12399 was merged 😀